route/src/route_generator_worker.cpp

```diff
@@ -358,48 +355,42 @@ namespace route {
             }
         }
 
-        route_marker_msg_.markers={};
+        route_marker_msg_.points={};
 
-        if (points.empty())
+        if (!points.empty())
         {
             ROS_WARN_STREAM("No central line points! Returning");
-            return route_marker_msg_;
```

Comment:
Why remove this return? It still seems valid since the route is empty

Reply:
Actually probably best to move this if statement to line 383 before the for loop so that a routing failure will clear the points from rviz